### PR TITLE
Bring back running pytorch tests in rocm CI

### PR DESCRIPTION
**Pull request by bddppq**

### CODE REVIEW

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -188,27 +188,22 @@ test_xla() {
...
+  test_python_all_except_nn
+  test_aten
+  test_libtorch
+  test_custom_script_ops
 else
-  if [[ "${BUILD_ENVIRONMENT}" == *-test1 ]]; then
-    test_torchvision
-    test_python_nn
-  elif [[ "${BUILD_ENVIRONMENT}" == *-test2 ]]; then
-    test_torchvision
```
Comment (bddppq) [inline .jenkins/pytorch/test.sh]: I think "test_torchvision" is duplicated here as test1 has already run it. @yf225

Comment (yf225) [reply .jenkins/pytorch/test.sh]: Yes it's likely a duplicate, thanks for catching it :D

**Review by yf225 — APPROVED** [review]

File: .jenkins/pytorch/test.sh (hunk 1 above)
Comment (aricheto) [reply .jenkins/pytorch/test.sh]: Test in h

Comment (aricheto) [reply .jenkins/pytorch/test.sh]: Test in h

**Pull request closed without merging**